ui/pages/confirmations/components/confirm/info/shared/edit-gas-fees-row/edit-gas-fees-row.tsx

```diff
@@ -33,10 +34,15 @@ export const EditGasFeesRow = ({
   const { useNativeCurrencyAsPrimaryCurrency: isNativeCurrencyUsed } =
     useSelector(getPreferences);
 
+  const transactionMeta = useSelector(
+    currentConfirmationSelector,
+  ) as TransactionMeta;
+
   return (
-    <ConfirmInfoRow
+    <ConfirmInfoAlertRow
+      alertKey="estimatedFee"
```

Comment:
We have the `RowAlertKey` enum for these so they can be referenced in the alert hooks.

Reply:
Done. https://github.com/MetaMask/metamask-extension/pull/26121/commits/32298a7b7bc2871747c0475546a47633766857e6